embedded-timestamp-mismatch/rewrite_embedded_bucket_id_control_min_mismatch.js

```diff
@@ -185,6 +188,28 @@ function shouldRetryTxnOnTransientError(e) {
   return false;
 }
 
+function checkValidateResForEmbeddedBucketIdControlMinMismatch(validateRes) {
+  return (validateRes.errors.length != 0 &&
+          validateRes.errors.some(x => x.includes('6698300'))) ||
+      (validateRes.warnings.length != 0 &&
+       validateRes.warnings.some(x => x.includes('6698300')));
+}
+
+function checkLogsForEmbeddedBucketIdControlMinMismatch() {
+  const getLogRes = db.adminCommand({getLog: 'global'});
+  if (getLogRes.ok) {
+    return getLogRes.log
+               .filter(
+                   line =>
+                       (line.includes('6698300') &&
+                        line.includes(mismatchEmbeddedIdTimestampMsg)))
+               .length > 0;
+  }
+  print(
+      '\ngetLog failed. Re-run checkLogsForEmbeddedBucketIdControlMinMismatch() or manually check mongodb logs for validation results.');
+  exit(1);
```

Comment:
I don't think we should `exit(1)` here, if getLog fails it would still be good to print the status of `checkValidateResForEmbeddedBucketIdControlMinMismatch`. It would be great if you could write this so only one error message is printed